symengine/tests/basic/test_fields.cpp

```diff
@@ -169,16 +184,42 @@ TEST_CASE("GaloisField Addition, Subtraction, Multiplication : Basic",
     REQUIRE(mp[5] == 3);
     REQUIRE(mp[6] == 1);
     REQUIRE(mp[7] == 2);
+
+    a = {3_z, 0_z, 0_z, 6_z, 1_z, 2_z};
+    b = {4_z, 0_z, 1_z, 0_z};
+    r1 = GaloisField::from_vec(x, a, 11_z);
+    r2 = GaloisField::from_vec(x, b, 11_z);
+    REQUIRE(r1->compare(*r2) == 1);
+    a = {3_z, 6_z, 1_z};
+    r1 = GaloisField::from_vec(x, a, 11_z);
+    REQUIRE(r1->compare(*r2) == -1);
+    r1 = GaloisField::from_vec(y, a, 11_z);
+    REQUIRE(r1->compare(*r2) == 1);
+    r1 = GaloisField::from_vec(x, a, 8_z);
+    REQUIRE(r1->compare(*r2) == -1);
 }
-TEST_CASE("GaloisFieldDict Division, GCD, LCM, Shifts : Basic", "[basic]")
+TEST_CASE("GaloisFieldDict Division, GCD, LCM, Shifts, Negation : Basic", "[basic]")
 {
     RCP<const Symbol> x = symbol("x");
-    std::vector<integer_class> a, b, mp;
-    GaloisFieldDict d1, d2, d3, d4;
+    std::vector<integer_class> a, b, c, mp;
+    GaloisFieldDict d1, d2, d3, d4, d5;
     a = {0_z, 1_z, 2_z, 3_z, 4_z, 5_z};
     b = {0_z, 3_z, 2_z, 1_z};
+    c = {};
     d1 = GaloisFieldDict::from_vec(a, 7_z);
     d2 = GaloisFieldDict::from_vec(b, 7_z);
+    d5 = GaloisFieldDict::from_vec(b, 5_z);
+    CHECK_THROWS_AS(d1.gf_div(d5, outArg(d3), outArg(d4)), SymEngineException &);
+    CHECK_THROWS_AS(d1.mul(d5, d1), std::runtime_error);
```

Comment:
```suggestion
    CHECK_THROWS_AS(d1.mul(d5, d1), std::runtime_error &);
```
(to avoid a compiler warning about catching by value that makes this build fail: https://travis-ci.org/github/symengine/symengine/jobs/746694646#L3220)

Reply:
Yes, thank you